.circleci/config.yml

```diff
@@ -156,10 +156,13 @@ jobs:
                     -p @0x/contracts-broker \
                     -p @0x/contracts-zero-ex \
                     -m --serial -c test:circleci
-    test-foundry:
+    test-foundry-local:
```

Comment:
#625 moves the tests to Github CI, so be prepared to migrate this over to make sure it doesn't just disappear.